README.md

```diff
@@ -126,10 +126,14 @@ The above would start the container using the configuration, call the `before` t
 
 ## Container Image (`from <string>`)
 
-The `from` configuration property instructs the image to be used on the primary instance and services.
+The `from` configuration property causes Binci to use the specified image to run tasks, rather than building a new image from the local Dockerfile.
 
 For testing different images easily, the `-f <alternate-image>` argument can be called during execution.
 
+Omitting this property will cause Binci to build a container from your project's Dockerfile. This container will be

```

Comment:
perhaps instead we assume an image pull, if the desire is to build, from goes from `from: some-image:latest` to `from: file://path/to/Dockerfile`.

Reply:
See top-level comment!